specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentralapps.json

```diff
@@ -7,15 +7,9 @@
   },
   "host": "apps.azureiotcentral.com",
   "basePath": "/api/preview",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
+  "schemes": ["https"],
```

Comment:
The formatting of this doc is wrong, please see https://aka.ms/AA6h31t for how to fix.

Reply:
got it, thanks